oneandone/resource_oneandone_server.go

```diff
@@ -526,6 +589,33 @@ func readIps(ips []oneandone.ServerIp) []map[string]interface{} {
 	return raw
 }
 
+func fixedsize2Id(config *Config, fixedsize string) (string, error) {
+	fixedsizes, err := config.API.ListFixedInstanceSizes()
+	if err != nil {
+		return "", fmt.Errorf("Could not fetch FixedInstanceSizes: %s ", err)
+	}
+	fixedsize = strings.ToLower(fixedsize)
+	for _, size := range fixedsizes {
+		if strings.ToLower(size.Name) == fixedsize {
+			return size.Id, nil
+		}
+	}
+	return "", fmt.Errorf("FixedInstanceName not found: %s ", fixedsize)
+}
```

Comment:
i tried adding this: ab9fe288d8d8c7930c923840f16883c5918c7b5c
but i can't get it to work:
```
data "oneandone_instance_size" "abc" {
        name = "L"
}
resource "oneandone_server" "serverA" {
  name                = "test-${data.oneandone_instance_size.abc.name}"
  description         = "test1"
  image               = "debian8-64min"
  datacenter          = "DE"
  fixed_instance_size  = "${data.oneandone_instance_size.abc.sid}"
}
```
results in an error:
```
oneandone_server.serverA: Resource 'data.oneandone_instance_size.abc' does not have attribute 'sid' for variable 'data.oneandone_instance_size.abc.sid'
```